### DISCUSSION TITLE
right clicking an app (or something else) in the bottom left menu should offer to show you where it is in the file explorer

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [linuxmint/discussions#700] [Ideas]
if i right click something, i want to see where it is located in my files

**Comment by Jopp-gh** [comment]
This can be done with a contextual menu script.
Check out :
https://forums.linuxmint.com/viewtopic.php?t=351366

You'll need the bash shell command `dirname`, see here:
https://linuxhandbook.com/dirname/
